src/types/dataOutput.ts

```diff
@@ -0,0 +1,19 @@
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import * as t from "io-ts";
+
+const DeduplicationStrategy = t.partial({
+  deduplicationStrategy: t.literal("TIMESTAMP")
```

Comment:
can deduplicationStrategy be an optional parameter ? In case it's not defined it should have a default value TIMESTAMP.